signer/src/testing/storage/model.rs

```diff
@@ -251,6 +255,14 @@ impl TestData {
 
         stacks_blocks
     }
+
+    /// Fetch the parent block given the hash.
+    pub fn get_parent_bitcoin_block(&self, parent: &BitcoinBlockHash) -> Option<BitcoinBlock> {
```

Comment:
Shouldn't this just be `get_bitcoin_block(block_hash)` since you're not actually doing any parent lookups here?

Reply:
Oh yeah, that's right, will update.

Reply:
https://github.com/stacks-network/sbtc/pull/540/commits/16c7b7d5b5c8ecab23c0bb435c0d02933c204de4